doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_memory.rst

```diff
@@ -232,54 +195,54 @@ Access control
+   The FLPR and its owner have access to all partitions assigned to FLPR and its Interprocessor Communication.
+   Each one of the memory partitions assigned for the DMA of the fast peripherals is accessible from the core owning the given set of peripherals.
    The FLPR and the fast peripherals are by default owned by the Application Core.
    This ownership and matching memory access rights can be customized in UICRs.
+   The FLPR has access to the entire RAM21 memory region.
+   Because of this property, the FLPR's owner indirectly obtains access to the entire RAM21 memory region.
+   To avoid security risks, all the partitions in RAM21 must be assigned to the FLPR's owner.
+   Also, all peripherals with DMA buffers in this memory must be assigned to the FLPR's owner.
+   The FLPR and the fast peripherals are by default owned by the Application Core.
+   This ownership and matching memory access rights can be reassigned to the Radio Core in the UICR.
 
-   The security attribute of memory partitions must follow FLPR and DMA engines security settings.
+   The security attribute of memory partitions DMA engines must follow the FLPR security settings.
 
 Slow global RAM (RAM3x)
 -----------------------
 
-.. image:: images/nrf54h20_memory_map_ram3x.svg
-   :width: 300 px
-
-The Slow global RAM is a part of the global RAM tightly coupled with the Peripheral Processor.
-Access to this memory block from the PPR and slow periperals' DMA has minimal latency and can be performed without powering up any other parts of the system.
+The Slow global RAM is a part of the global RAM close to the Peripheral Processor.
+Access to this memory block from the PPR and slow peripherals' DMA has minimal latency and can be performed without powering up any other parts of the system.
 Access to this memory from the local domains has higher latency than access to the general-purpose shared RAM.
 
 This memory is intended to store the code executed in the PPR, the PPR's data, the shared memory used for Inter-Processor Communication between the PPR and the core managing the PPR, and DMA buffers for the slow peripherals.
 
 Address range
-   0x2FC00000 - 0x2FC08000
+   0x2FC00000 - 0x2FC14000
 
 Size
-   32 KB
+   80 KB
 
 Access control
   The PPR and its owner have access to all the partitions assigned to the PPR and its Inter-Processor Communication.
   Each of the memory partition assigned for DMA of the slow peripherals is accessible from the core owning the given set of peripherals.
   The PPR and the slow peripherals are by default owned by the Application Core.
   The ownership and matching memory access rights can be customized in UICRs.
 
-  The security attribute of memory partitions must follow PPR and CMA engines security settings.
+  The security attribute of memory partitions CMA engines must follow the PPR security settings.
```

Comment:
The previous statement was correct. It's different here for RAM_3x than it was in RAM_21 description. The difference is the result of RAM_21 being actually tightly coupled to FLPR, while RAM_3x is not coupled to PPR.

```suggestion
  The security attribute of memory partitions must follow PPR and DMA engines' security settings.
```